en/docs/chapter_sorting/selection_sort.md

```diff
@@ -51,7 +51,7 @@ In the code, we use $k$ to record the smallest element within the unsorted inter
 
 ## Algorithm characteristics
 
-- **Time complexity of $O(n^2)$, non-adaptive sort**: There are $n - 1$ rounds in the outer loop, with the unsorted interval length starting at $n$ in the first round and decreasing to $2$ in the last round, i.e., the outer loops contain $n$, $n - 1$, $\dots$, $3$, $2$ inner loops respectively, summing up to $\frac{(n - 1)(n + 2)}{2}$.
+- **Time complexity of $O(n^2)$, non-adaptive sort**: There are $n - 1$ iterations in the outer loop, with the unsorted section length starting at $n$ in the first iteration and decreasing to $2$ in the last iteration, i.e., the outer loop iterations contain $n$, $n - 1$, $\dots$, $3$, $2$ inner loop iterations respectively, summing up to $\frac{(n - 1)(n + 2)}{2}$.
```

Comment:
_**each**_ outer loop iterations····
Including the word “each” clarifies that every individual outer loop iteration corresponds to a specific number of inner loop iterations, enhancing the sentence's clarity.